frontend-react/src/network/Apis.ts

```diff
@@ -9,12 +9,17 @@ interface ReportStreamApi {
     basePath: string;
     headers: AxiosRequestHeaders;
     root: string;
-    updateSession: (headers: AxiosRequestHeaders) => void;
+    updateSession: (headers: ReportStreamApiHeaders) => void;
 }
 
 const apis: ReportStreamApi[] = [];
 
-const headersFromStoredSession = (): AxiosRequestHeaders => ({
+interface ReportStreamApiHeaders {
```

Comment:
Change 2: `AxiosRequestHeaders` is no longer compatible with the return value from `headersFromStoredSession`.  I just added a custom interface here since its usage is pretty limited, but open to other suggestions!